### PR TITLE
Stock Prices Watcher demo app

### PR DESCRIPTION
**Pull request by maxdml**

### CODE REVIEW

**Comment by apoliakov** [comment]
Hey quick question - does this emit a trace when users visit the app?
For example when it runs the stock query can it emit an OTEL trace? 
Would really help with monitoring and serve as a good example.

**Comment by maxdml** [comment]
> Hey quick question - does this emit a trace when users visit the app?

No, this app has no HTTP endpoints. It just starts a scheduled job in the backend. And because we use streamlit for the UI, we can't use DBOS endpoints (which generate OTel traces) to serve requests (streamlit reloads the entire code every time you interact, and doesn't launch DBOS, etc)

**Review by kraftp — COMMENTED** [review]
Looks good!  Let's format this with `isort` and `black` to get it in line with the others

File: python/stock-prices/stock_prices/main.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+def send_sms_alert(symbol, price, to_phone_number):
+    client = Client(twilio_account_sid, twilio_auth_token)
+    message = client.messages.create(
+        body=f"{symbol} stock price is {price}.",
+        from_=twilio_phone_number,
+        to=to_phone_number
+    )
+    DBOS.logger.info(f"SMS sent: {message.sid}")
+
+# Let's write a small function to retrieve alerts from the database
```
Comment (kraftp) [inline python/stock-prices/stock_prices/main.py:53]: Where do the alerts come from?

**Comment by maxdml** [comment]
> Looks good! Let's format this with `isort` and `black` to get it in line with the others

done

File: python/stock-prices/stock_prices/streamlit.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
+# Stock prices
+
+# This app uses DBOS to periodically fetch stock prices from Yahoo Finance, store them into Postgres
+# and display them using Streamlit.
+
+# This part of the app uses Streamlit for data visualization.
+# The ingestion code using DBOS Transact is in main.py.
+# We separate the Streamlit script from the DBOS ingestion code
+# because Streamlit re-runs the entire script every time it's viewed.
+
```
Comment (kraftp) [inline python/stock-prices/stock_prices/streamlit.py:10]: Maybe make "stock prices" the default view as that's more comprehensible?

**Review by kraftp — APPROVED** [review]
Looks good!

**Pull request merged**